docker/run_celery.bash

```diff
@@ -7,11 +7,14 @@ source /etc/profile
 # according to the user's preference saved by django-constance
 cd "${KPI_SRC_DIR}"
 
+AUTOSCALE_MIN="${CELERY_AUTOSCALE_MIN:-2}"
+AUTOSCALE_MAX="${CELERY_AUTOSCALE_MAX:-2}"
```

Comment:
Let's make it 6 max then. This setting isn't useful for k8s and is fine IMO as a default elsewhere. The extra ram usage if there are 4 instead of 6 cores is minimal and anyone can override it if they need to. I think this will be more obvious anyway, it certainly gave me the impression it was set to just 2.